### ISSUE TITLE
Instructions are obsolete. Everything changed. 

### ISSUE THREAD

**Issue by MarcSchluperAtIntel** [SharePoint/sp-dev-docs#9869]
Somebody changed everything on https://entra.microsoft.com/#view/Microsoft_AAD_IAM/TenantOverview.ReactView

Sigh.

Just like at Trader Joe's. New store manager, who insists everything needs to be displaced. And now we can all try to find what was obvious before.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 80e62a9d-06d0-ebc3-2c99-ce459388182a
* Version Independent ID: 07d8b090-b18c-842d-ba1e-0f2b2a127ad3
* Content: [Using CSOM for .NET Standard instead of CSOM for .NET Framework](https://learn.microsoft.com/en-us/sharepoint/dev/sp-add-ins/using-csom-for-dotnet-standard)
* Content Source: [docs/sp-add-ins/using-csom-for-dotnet-standard.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/sp-add-ins/using-csom-for-dotnet-standard.md)
* Service: **sharepoint**
* GitHub Login: @VesaJuvonen
* Microsoft Alias: **vesaj**

**Comment by Ashlesha-MSFT** [comment]
Hello @MarcSchluperAtIntel,
Thank you for bringing this issue to our attention. We will look into it and get back to you shortly.

**Comment by Ashlesha-MSFT** [comment]
@MarcSchluperAtIntel,
Yes, Microsoft has recently made significant UI and structural changes to the Entra (formerly Azure AD) portal at https://entra.microsoft.com/. Key sections like app registrations, API permissions, and authentication settings have been reorganized under the new Entra Identity structure.

While all core functionality remains available, many settings are now located in different sections or under new labels, so navigation requires adjustment. If you're looking for a specific setting or flow, let me know and I can help direct you to the updated location.

**Comment by Ashlesha-MSFT** [comment]
Thank you for the feedback.
The behavior described is related to recent UI changes in the Microsoft Entra admin center, but these updates do not affect the accuracy or validity of the documentation for Using CSOM for .NET Standard instead of CSOM for .NET Framework.

The article focuses on CSOM authentication, .NET Standard guidance, and SharePoint Add-in deprecation and does not rely on the specific Entra portal layout.

No documentation update is required at this time.

We are closing this issue, but please feel free to reopen or submit a new issue if you encounter any documentation inaccuracies.
